api/environments/dynamodb/migrator.py

```diff
@@ -81,4 +82,5 @@ def migrate(self):
             )
         )
         identity_wrapper.write_identities(iterator_with_prefetch(identities))
+        migrate_project_environments_to_v2(project_id)
```

Comment:
This will need more work since it won't work unless we set the read capacity budget against the project. We should therefore just write the identity overrides directly to the `flagsmith_environments_v2` table but this requires a larger refactor of the logic.